convert2rhel/main.py

```diff
@@ -158,13 +158,9 @@ def main_locked():
             subscription.update_rhsm_custom_facts()
 
         rollback_changes()
+        print_info_after_rollback(pre_conversion_results, True)
 
-        report.summary(
-            pre_conversion_results,
-            include_all_reports=True,
-            disable_colors=logger_module.should_disable_color_output(),
-        )
-        return 0
+        return backup.backup_control.rollback_failed
```

Comment:
This will return `False` instead of 1, right?

```
/tmp> python t.py                                                                                                                             03/22/24 09:27:19 AM
/tmp> $env.LAST_EXIT_CODE                                                                                                                     03/22/24 09:27:20 AM
0
/tmp> cat t.py                                                                                                                                03/22/24 09:27:23 AM
def test():
    return False

test()
```

Reply:
And if it is 0, why not keeping it simply as 0?

Reply:
Ah, this test is false... It will always return the exit code for the interpreter instead of the return code of the script. To make it return the exit code for the script, we would need to use sys.exit() passing the return value for the function to it